### PR TITLE
eliminate the race to read the serving certs in operator filepath change detection

### PR DESCRIPTION
**Pull request by deads2k**
We knew we had a theoretical race, but we didn't expect to see it happen in any practical timeframe.  This is a good candidate fix for https://bugzilla.redhat.com/show_bug.cgi?id=1678929 which appears to happen sometimes.  We just force the initial values to hash.

/hold

hold for proof in another pull.

### CODE REVIEW

File: pkg/controller/controllercmd/cmd.go [hunk 1]
```diff
@@ -146,10 +153,23 @@ func (c *ControllerCommandConfig) StartController(ctx context.Context) error {
...
+			if err := servingCert.WriteCertConfigFile(config.ServingInfo.CertFile, config.ServingInfo.KeyFile); err != nil {
+				return err
+			}
+			crtContent := &bytes.Buffer{}
+			keyContent := &bytes.Buffer{}
+			if err := servingCert.WriteCertConfig(crtContent, keyContent); err != nil {
+				return err
+			}
+
+			// If we generate our own certificates, then we want to specify empty content to avoid a starting race.  This way,
```
Comment (bparees) [inline pkg/controller/controllercmd/cmd.go]: comment says "specify empty content" but this doesn't look like empty content, what am i missing?

Comment (deads2k) [reply pkg/controller/controllercmd/cmd.go]: > comment says "specify empty content" but this doesn't look like empty content, what am i missing?

stale comments.  clearly I should not write comments :)

**Comment by bparees** [comment]
@adambkaplan we need to make sure the openshift controller manager operator picks up a bump w/ this change after it merges.  (my other bump PR does not include this).

**Comment by deads2k** [comment]
it's no worse anyway.

**Pull request merged**